pkg/simple/client/metering/options.go

```diff
@@ -1,7 +1,41 @@
 package metering
 
+type PriceInfo struct {
+	// currency unit, currently support CNY and USD
+	CpuPerCorePerHour float64 `json:"cpuPerCorePerHour" yaml:"cpuPerCorePerHour"`
+	// cpu cost with above currency unit for per core per hour
+	MemPerGigabytesPerHour float64 `json:"memPerGigabytesPerHour" yaml:"memPerGigabytesPerHour"`
+	// mem cost with above currency unit for per GB per hour
+	IngressNetworkTrafficPerMegabytesPerHour float64 `json:"ingressNetworkTrafficPerMegabytesPerHour" yaml:"ingressNetworkTrafficPerGiagabytesPerHour"`
+	// ingress network traffic cost with above currency unit for per MB per hour
+	EgressNetworkTrafficPerMegabytesPerHour float64 `json:"egressNetworkTrafficPerMegabytesPerHour" yaml:"egressNetworkTrafficPerGigabytesPerHour"`
+	// egress network traffice cost with above currency unit for per MB per hour
+	PvcPerGigabytesPerHour float64 `json:"pvcPerGigabytesPerHour" yaml:"pvcPerGigabytesPerHour"`
+	// pvc cost with above currency unit for per GB per hour
+	CurrencyUnit string `json:"currencyUnit" yaml:"currencyUnit"`
```

Comment:
Normally the comments are above from the code lines. Please see also the introduction from [Golang blog](https://blog.golang.org/godoc).